cpp/oneapi/dal/backend/interop/common.hpp

```diff
@@ -46,21 +46,14 @@ struct to_daal_cpu_type<cpu_dispatch_avx512> : daal_cpu_value<daal::avx512> {};
 
 inline constexpr cpu_extension from_daal_cpu_type(daal::CpuType cpu) {
     switch (cpu) {
-        case daal::sse2:
-            return cpu_extension::sse2;
-        case daal::ssse3:
-            return cpu_extension::ssse3;
-        case daal::sse42:
-            return cpu_extension::sse42;
-        case daal::avx:
-            return cpu_extension::avx;
-        case daal::avx2:
-            return cpu_extension::avx2;
-        case daal::avx512:
-            return cpu_extension::avx512;
+        case daal::sse2: return cpu_extension::sse2;
+        case daal::ssse3: return cpu_extension::ssse3;
+        case daal::sse42: return cpu_extension::sse42;
+        case daal::avx: return cpu_extension::avx;
+        case daal::avx2: return cpu_extension::avx2;
+        case daal::avx512: return cpu_extension::avx512;
         case daal::avx512_mic:
```

Comment:
Is it correct not to return anything? If yes why do this lines are presented here?